carts/views.py

```diff
@@ -44,3 +44,26 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message" : "KEY_ERROR"}, status=400)
 
+    @login_decorator
+    def patch(self, request, cart_id):
+            data      = json.loads(request.body)
+            quantity  = data['quantity']
+            
+            if quantity <= 0:
+                return JsonResponse({'message':'QUANTITY_ERROR'}, status=400)
+
+            cart = Cart.objects.get(id=cart_id, user=request.user)
```

Comment:
get method에 대한 예외 처리(DoseNotExist 에러) 추가해 주세요.
```suggestion
            cart = Cart.objects.get(id=cart_id, user=request.user)
```